release-1.8/release_notes_draft.md

```diff
@@ -317,69 +318,65 @@ kind.
- replace defaulted values.
+ RollingUpdate for the apps/v1beta2 group version. Users can explicitly set
+ the OnDelete strategy, and no strategy auto-conversion is applied to
+ replace default values.
  - As mentioned in [Behavioral Changes](#behavioral-changes), selector
- defaulting is disabled.
+ defaults are disabled.
  - The default spec.revisionHistoryLimit for all applicable kinds in the
- apps/v1beta2 group version has set to 10.
+ apps/v1beta2 group version is 10.
  - The default spec.successfulJobsHistoryLimit is 3 and spec.failedJobsHistoryLimit
    is 1 on CronJobs.
 
-### [Workload API (batch)]
-- CronJob has been promoted to `batch/v1beta1` ([#41039](https://github.com/kubernetes/kubernetes/issues/41039), [@soltysh](https://github.com/soltysh)).
-- `batch/v2alpha.CronJob` has been deprecated in favor of `batch/v1beta` and will be removed in future releases.
-- Job has now the ability to set a failure policy using `.spec.backoffLimit`.  The default value for this new field is set to 6. ([#30243](https://github.com/kubernetes/kubernetes/issues/30243), [@clamoriniere1A](https://github.com/clamoriniere1A)).
-- `batch/v2alpha1.ScheduledJob` has been removed.
-- Job controller creates pods in batches instead of all at once ([#49142](https://github.com/kubernetes/kubernetes/pull/49142), [@joelsmith](https://github.com/joelsmith)).
-- Short `.spec.ActiveDeadlineSeconds` is properly applied to a job ([#48545](https://github.com/kubernetes/kubernetes/pull/48454), [@weiwei4](https://github.com/weiwei04)).
+### Workloads API (batch)
+- CronJob is now at `batch/v1beta1` ([#41039](https://github.com/kubernetes/kubernetes/issues/41039), [@soltysh](https://github.com/soltysh)).
+- `batch/v2alpha.CronJob` is deprecated in favor of `batch/v1beta` and will be removed in a future release.
+- Job can now set a failure policy using `.spec.backoffLimit`. The default value for this new field is 6. ([#30243](https://github.com/kubernetes/kubernetes/issues/30243), [@clamoriniere1A](https://github.com/clamoriniere1A)).
+- `batch/v2alpha1.ScheduledJob` is removed.
+- The Job controller now creates pods in batches instead of all at once. ([#49142](https://github.com/kubernetes/kubernetes/pull/49142), [@joelsmith](https://github.com/joelsmith)).
+- Short `.spec.ActiveDeadlineSeconds` is properly applied to a Job. ([#48545](https://github.com/kubernetes/kubernetes/pull/48454), [@weiwei4](https://github.com/weiwei04)).
 
 
 #### CLI Changes
 
-- [alpha] `kubectl` plugins: `kubectl` now allows binary extensibility as an alpha
-  feature. Users can extend the default set of `kubectl` commands by writing plugins
-  that provide new subcommands. Please refer to the documentation for more information.
+- [alpha] `kubectl` plugins: `kubectl` now allows binary extensibility. Users can extend the default set of `kubectl` commands by writing plugins
+  that provide new subcommands. Refer to the documentation for more information.
```

Comment:
provide a link to the documentation.

Reply:
Yeah, I don't have a link to the new stuff yet. I'll keep working in this PR and ping Steve when he comes online.

Reply:
Still looking for appropriate doc link. Can we merge without it? More changes still TK for workloads 1.8->1.9 stuff.